projects/examples/src/components/datagrid/datagrid-row-icon.example.component.ts

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component, Input } from '@angular/core';
+import {
+    GridColumn,
+    GridDataFetchResult,
+    GridState,
+    ComponentRenderer,
+    ColumnComponentRendererSpec,
+    ColumnRendererSpec,
+} from '@vcd/ui-components';
+
+interface Data {
+    href: string;
+}
+
+@Component({
+    selector: 'vcd-datagrid-row-icon-example',
+    template: `
+        <vcd-datagrid [gridData]="gridData" (gridRefresh)="refresh($event)" [columns]="columns"></vcd-datagrid>
```

Comment:
Isn't this example showing that rows maintain their state when re-rendered? Please add text to the top of the grid to help users understand what the example is trying to show.